bids_prov/fsl/fsl_parser.py

```diff
@@ -247,12 +383,15 @@ def build_records(groups: Mapping[str, List[str]], agent_id: str):
             for df in description_functions:
                 if df["name"] == command_name_end:
                     function_in_description_functions = True
-                    if "used" in df:
-                        entities, _ = get_entities(cmd_s, df["used"])
-                        inputs.extend(entities)
-                    if "generatedBy" in df:
-                        entities, _ = get_entities(cmd_s, df["generatedBy"])
-                        outputs.extend(entities)
+                    inputs, outputs, parameters = get_entities(cmd_s[1:], df)
```

Comment:
not explicit that df called in get_entities is the one from command_name_end. Perhaps add and explicit variable name

Reply:
FIX in commit **answer hermann **

![image](https://user-images.githubusercontent.com/56917204/224978818-d527eb2e-9a15-4fd6-81ce-a0425d4b012a.png)